cocotb/share/makefiles/Makefile.pylib.Msys

```diff
@@ -47,3 +47,20 @@ ifeq ($(PYTHON_LIBDIR),None)
 endif
 
 PYTHON_INCLUDEDIR := $(shell $(PYTHON_BIN) -c 'import distutils.sysconfig; print( distutils.sysconfig.get_python_inc() )')
+
+# For tcl scripts purpose source files paths in form of C:/User/..
+ifneq ($(VERILOG_SOURCES),)
+    VERILOG_SOURCES := $(shell cygpath -m $(VERILOG_SOURCES))
+endif
+ifneq ($(VHDL_SOURCES),)
+    VHDL_SOURCES := $(shell cygpath -m $(VHDL_SOURCES))
+endif
+
+# On Windows there is no support for @rpath so we need to set PATH to find cocotb libraries
+export PATH := $(LIB_DIR):$(PATH)
+
+# Set PYTHONHOME to properly populate sys.path in embedded python interpreter
+export PYTHONHOME := $(shell $(PYTHON_BIN) -c "from distutils.sysconfig import get_config_var; print(get_config_var('prefix'))")
```

Comment:
I think we want to do one of two things:

* Eliminate `PYTHON_BIN`, and use something from `cocotb-config` instead
* Eliminate `cocotb-config`, and use `$(PYTHON_BIN) -m cocotb.config` instead

Otherwise, we can end up mixing two different python installations.

Reply:
Yes. It was not available at the time. I can rebase and clean this up a bit now. I like `Middle-ground`.

Reply:
In the long run, I'm wondering if eliminating `cocotb-config` is the way to go.

Reply:
Long run eliminating Makefiles maybe ;-)  ?

Reply:
This line is about PYTHONHOME, not PYTHON_BIN. PYTHON_BIN can already be set from `cocotb-config --python-bin`, we might want to to the same for PYTHONHOME, or keep the line of Python as it is in this patch.

Reply:
I will prepare separate PR.